# **0. 함수란?**

* 함수는 이름 뒤에 소괄호
* 함수는 자주 쓰는 기능을 간편하게 모아 놓은 것이다

# 1. **print() 함수**

소괄호 안에 있는 값을 출력해주는 기능을 한다.

In [2]:
print('hello')

hello


In [4]:
print(3.14)

3.14


# **2. 값의 종류**

값의 종류는 **type** 이라고 부르기도 하며, **자료형** 이라고 부르기도 한다.

*   **int**

> 10, 20, -50, ....(정수)

*   **float**

> 3.1, -2.4, ....(실수)

*   **string**

> '안녕하세요', '반갑습니다', 'apple', ....(문자열)

*   **bool**
*   **list**
*   **tuple**
*   **dict**
*   **set**

